projects/packages/my-jetpack/_inc/components/product-cards-section/backup-card/index.jsx

```diff
@@ -128,12 +130,58 @@ const getTimeSinceLastRenewableEvent = lastRewindableEventTime => {
 const BackupCard = props => {
 	const { detail } = useProduct( productSlug );
 	const { status } = detail;
+	const { backup_failure: backupFailure } =
+		getMyJetpackWindowInitialState( 'redBubbleAlerts' ) || {};
+	const { status: lastBackupStatus } = backupFailure || {};
 	const hasBackups = status === PRODUCT_STATUSES.ACTIVE || status === PRODUCT_STATUSES.CAN_UPGRADE;
+	const noDescription = () => null;
 
-	return hasBackups ? (
-		<WithBackupsValueSection slug={ productSlug } { ...props } />
-	) : (
-		<ProductCard slug={ productSlug } { ...props } />
+	const { title: errorTitle, text: errorDescription } = useGetReadableFailedBackupReason() || {};
+
+	if ( hasBackups ) {
+		return <WithBackupsValueSection slug={ productSlug } { ...props } />;
+	}
+
+	return (
+		// eslint-disable-next-line react/jsx-no-bind
+		<ProductCard slug={ productSlug } Description={ noDescription } { ...props }>
+			{ status === PRODUCT_STATUSES.NEEDS_ATTENTION && backupFailure && (
+				<div className={ styles.backupErrorContainer }>
+					<div className={ styles.iconContainer }>
+						<Gridicon icon="notice" size={ 16 } className={ styles.iconError } />
+					</div>
+					<div className={ styles.contentContainer }>
+						<Text variant="body-small" className="value-section__heading">
+							{ __( 'The last backup attempt failed.', 'jetpack-my-jetpack' ) }
+							<InfoTooltip
+								tracksEventName={ 'protect_card_tooltip_open' }
+								tracksEventProps={ {
+									location: 'backup-error',
+									status: status,
+									backup_status: lastBackupStatus,
+									feature: 'jetpack-backup',
+									message: '', //errorDescription,
```

Comment:
Can we remove that commented out variable by chance?

Reply:
Good catch thanks!